### DISCUSSION TITLE
PSA - OctoPrint 1.6 Update

### DISCUSSION THREAD

**Discussion by Timbo200** [Jyers/Marlin#636] [General Discussion]
I'm not sure if this is related to the communication issue we've been having between OctoPrint and what appears to be Marlin / Meatpack issue.

I had a stable connection on 1.5 and the printer was working fine and preheating.  Since I didn't mind if I was going to have to delay printing or there were 1.6 issues I gave the OK to update from 1.5 to 1.6.  Update installed and but afterwards OctoPrint hung.  I was able to access the API, but not the UI via a web browser.  So parts of OctoPrint were working.

A quick Google suggested that communication issues with the printer caused similar issues in the past so I figured what the heck and powered off the printer and used the API via my Android application  to reboot the Pi.  I'd imagine you could use SSH as well.  Pi rebooted with the printer off and the UI came back up.

TLDR - power the printer off before you upgrade.

**Comment by darco415** [comment]
Updated Octoprint earlier today. Meatpack and Octo working fine
